webpack.config.js

```diff
@@ -29,21 +29,19 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.(ts|js)?$/,
+        exclude: /\/node_modules\//,
+        test: /\.ts$/,
         use: {
           loader: 'esbuild-loader',
-          options: {
-            loader: 'ts',
-            target: 'chrome67'
-          }
+          options: { loader: 'ts' }
         }
       }
     ]
   },
   optimization: {
     minimizer: [
       new ESBuildMinifyPlugin({
-        target: 'chrome67' // Syntax to compile to (see options below for possible values)
```

Comment:
I'm not sure if we should specify a target in the loader or minifier, I'll trust that this is the correct place, but it cannot be es2015. The exponentiation operator will be transformed and break bigints https://esbuild.github.io/content-types/#javascript We should keep it chrome67

Reply:
Ok, very well.